docs/build/circuits.ipynb

```diff
@@ -97,7 +103,7 @@
     "id": "77zB_UqkAwCR"
    },
    "source": [
-    "![Circuits and Moments](/cirq/images/CircuitMomentOperation.png)"
+    "![Circuits and Moments](https://github.com/quantumlib/Cirq/blob/master/docs/images/CircuitMomentOperation.png)"
```

Comment:
Please replace (here and below) with a relative link, e.g., `(../images/CircuitMomentOperation.png)`

Reply:
Done